gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright � 2015-2021 the original authors.
+# Copyright � 2015-2021 the original authors.
```

Comment:
Don't change the gradle wrapper

Reply:
Or at least use reasonable replacements for the extended characters in the comment